index.js

```diff
@@ -0,0 +1,84 @@
+const Base = require('bfx-facs-base')
+const { HyperCoreLogger } = require('hypercore-logs')
+const HyperCoreTransport = require('./lib/winston-transport-hypercore')
+const path = require('path')
+const winston = require('winston')
+
+class HyperCoreLoggerFacility extends Base {
+  constructor (caller, opts, ctx) {
+    super(caller, opts, ctx)
+
+    this.name = 'hypercore-logger'
+    this._hasConf = true
+
+    this.init()
+  }
+
+  _start0 (cb) {
+    super._start0(err => {
+      if (err) {
+        return cb(err)
+      }
+
+      const hypercoreCfg = this.conf
+      if (!hypercoreCfg?.enable) {
+        return cb(null)
+      }
+
+      const feedDir = path.join(process.cwd(), hypercoreCfg.storageDir)
```

Comment:
this doesn't seem right to me as what if someone puts absolute path? I believe we could use something like this https://github.com/bitfinexcom/hypercore-logs/blob/master/src/helper.js#L12